packages/gatsby-source-shopify/src/query-builders/locations-query.ts

```diff
@@ -49,12 +49,7 @@ export class LocationsQuery extends BulkQuery {
                 handle
                 id
                 inventoryManagement
```

Comment:
This was apparently removed in version `2024-07` but cannot test at the moment because the API is returning `null` for `fulfillmentService`